src/index.js

```diff
@@ -51,6 +53,7 @@ async function loopWindows(extensionAPI) {
           // console.log(sidebar[i])
           removeWindow(sidebar[i])
         }
+        window.roamAlphaAPI.ui.rightSidebar.close();
```

Comment:
In the version in prod right now, what happens is that all windows are removed, which closes the sidebar automatically. Then when sidebar is reopened the pinned windows are resurrected

The filter above in line 46 means that not all windows will be removed. So, we have to manually close the right sidebar to maintain same behavior as before